internal/pkg/utils/payload_test.go

```diff
@@ -0,0 +1,35 @@
+package utils
+
+import (
+	"testing"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCheckPayloadSize(t *testing.T) {
+	payload := make([]byte, 10)
+	tests := []struct {
+		name            string
+		sizeLimit       int64
+		errorExpected   bool
+		expectedErrKind errors.ErrKind
+	}{
+		{"Valid Size", int64(len(payload)), false, ""},
+		{"Invalid Size", int64(len(payload) - 1), true, errors.KindLimitExceeded},
```

Comment:
add the invalid case for the payload size is larger than the default 25kb

Reply:
Do you mean we use 25kb here?
https://github.com/edgexfoundry/edgex-go/blob/dc6b798737b202c23945748eed2b99c1986bd439/internal/pkg/utils/payload_test.go#L12